packages/sync/src/modules/Terms.php

```diff
@@ -11,6 +11,14 @@ function name() {
 		return 'terms';
 	}
 
+	public function get_object_by_id( $object_type, $id ) {
```

Comment:
I'm not sure why we need the `$object_type` argument - is it because we're planning to hook it somewhere where we can't specify the arguments? I might be wrong, but I'm expecting that method won't be called by a hook, and if that's the case, we should be able to omit the type? I mean, we already know the type if we're calling this method, because it's defined in the terms class 😉 

Let me know if I'm missing something though - I assume you did that for a good reason.

Reply:
I'm not sure why either, but it's the parents abstract method, so we'll need to follow that pattern.